src/oauth/oauth2.js

```diff
@@ -78,20 +79,23 @@ export default class OAuth2 {
     let payload = objectExtend({}, userData)
 
     for (let key in defaultProviderConfig.responseParams) {
-      let value = defaultProviderConfig[key]
+      let value = defaultProviderConfig.responseParams[key]
```

Comment:
Actually, we shouldn't use `responseParams` from `defaultProviderConfig` but from `this.providerConfig` instead. That way we could pass `responseParams` when setting up `vue-authenticate` provider config which will be included in the `exchangeForToken` request. I believe that this approach would make much more elegant solution for the reported issue.

Reply:
addressed in https://github.com/dgrubelic/vue-authenticate/pull/139/files#diff-c3849faba68f56a667cf75ea2c02ea81